pkg/cgroups/v2path.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Intel Corporation. All Rights Reserved.
```

Comment:
Maybe we should update the copyright year :smile:

Reply:
That code was written last year... ;-)

Reply:
But I'll update anyway...